### PR TITLE
fix: nushell br alias fails on alt-enter if folder contains spaces

### PR DESCRIPTION
**Pull request by FrancescElies**
This is tiny fix for the case where a path contains white spaces in it.

Maybe @mediumrarez could double check what I am proposing is fine too?

### CODE REVIEW

**Comment by mediumrarez** [comment]
It looks fine for the most part, but we should remove the `str replace -a '"' ''` line. Nushell is smart enough to deal with quotes by itself. It'll just cause path not found on linux when the path has `"` in in it.

**Comment by FrancescElies** [comment]
@mediumrarez I agree with you, I couldn't test the `"` case (currently in a win. machine and it does not allow that character in the path) but I added a comment of how to test that code when doing changes to it.

Can you give it a try on your side?

**Comment by Canop** [comment]
@FrancescElies You can add commits, no need to force push. I'll squash when merging if it's messy.

**Comment by FrancescElies** [comment]
I am used to remove stuff in the history that adds nothing, but if you are checking out the branch you are right is more comfortable for you to just pull, I'll do like that from now on.

File: src/shell_install/nushell.rs [hunk 1]
```diff
@@ -14,15 +14,10 @@
...
-        conf,
-        errors::*,
-    },
+    crate::{conf, errors::*},
     directories::BaseDirs,
     std::path::PathBuf,
-    termimad::{
-        mad_print_inline,
-    },
+    termimad::mad_print_inline,
```
Comment (FrancescElies) [inline src/shell_install/nushell.rs:20]: @Canop I had autoformat with cargo activated and I changed this lines by accident, this has nothing to do with this pr, shall I revert it or leave it as it is?

Comment (Canop) [reply src/shell_install/nushell.rs:20]: It's Ok, don't worry for such a small change, don't lose time reverting.

**Comment by mediumrarez** [comment]
We could probably expand the regex to handle no-quotes and single quotes too. (Tested on Windows.)

```nushell
let target_dir = (open $cmd_file | parse -r `^cd\s+(?<quote>"|'|)(?<path>.+)\k<quote>[\s\r\n]*$` | get path | to text)
```

I'll have to test on linux tomorrow; it's too late for tonight.

File: src/shell_install/nushell.rs (hunk 1 above)
Comment (Canop) [reply src/shell_install/nushell.rs:20]: BTW I'd like to use cargo fmt but things are slow to stabilize...
```
Warning: can't set `imports_layout = Vertical`, unstable features are only available in nightly channel.
Warning: can't set `version = Two`, unstable features are only available in nightly channel.
```

**Comment by FrancescElies** [comment]
I will try it on Monday, but it looks good to me too, if you don’t mind shall I integrate it into this pr?

**Comment by mediumrarez** [comment]
No, go ahead. Public domain 👌.

**Comment by FrancescElies** [comment]
@mediumrarez if you are fine with the current code so am I :) and we could proceed for the merge

**Comment by mediumrarez** [comment]
@FrancescElies Looks OK to me.

**Comment by Canop** [comment]
OK, so I'll merge and try to have the new script replace the older one, right ?

**Comment by FrancescElies** [comment]
I would say that's the right thing to do

**Comment by Canop** [comment]
I'll add a system to have the script update proposed.

**Comment by Canop** [comment]
https://github.com/Canop/broot/pull/655

**Pull request merged**